File: src/main/java/ru/tykvin/hermes/configuration/SimpleMessageConvertersConfiguration.java
package ru.tykvin.hermes.configuration;

import lombok.RequiredArgsConstructor;
import org.springframework.context.annotation.Configuration;
import org.springframework.http.converter.FormHttpMessageConverter;
import org.springframework.http.converter.HttpMessageConverter;
import org.springframework.http.converter.StringHttpMessageConverter;
import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
import ru.tykvin.hermes.lib.JsonUtils;

import java.util.List;

@Configuration
@RequiredArgsConstructor
public class SimpleMessageConvertersConfiguration implements WebMvcConfigurer {

    @Override
    public void configureMessageConverters(List<HttpMessageConverter<?>> converters) {
        converters.add(new FormHttpMessageConverter());
        converters.add(new MappingJackson2HttpMessageConverter(JsonUtils.getObjectMapper()));
        converters.add(new StringHttpMessageConverter());
    }

}

File: src/main/kotlin/ru/tykvin/hermes/configuration/SimpleMessageConvertersConfiguration.kt
package ru.tykvin.hermes.configuration

import com.fasterxml.jackson.databind.ObjectMapper
import org.springframework.context.annotation.Configuration
import org.springframework.http.converter.FormHttpMessageConverter
import org.springframework.http.converter.HttpMessageConverter
import org.springframework.http.converter.StringHttpMessageConverter
import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

@Configuration
open class SimpleMessageConvertersConfiguration(
        private val om: ObjectMapper
) : WebMvcConfigurer {

    override fun configureMessageConverters(converters: MutableList<HttpMessageConverter<*>>) {
        converters.add(FormHttpMessageConverter())
        converters.add(MappingJackson2HttpMessageConverter(om))
        converters.add(StringHttpMessageConverter())
    }

}